a-hackathon-for-panthers-pantherhacks.md

```diff
@@ -0,0 +1 @@
Redeem those points for Compi tokens that will allow them to purchase tech equipment.” [![picture9](http://pantherhackers.com/wp-content/uploads/2016/11/Picture9-300x200.jpg)](http://pantherhackers.com/wp-content/uploads/2016/11/Picture9.jpg) Thanks to everyone who volunteered and competed in PatherHacks! Here at PantherHackers, we like to reflect on what we have done, what we have accomplished, and what we can do better in the future. We sat down with PantherHackers’s Senior Advisor, Tiffany Green-Abdullah, in order to see what we as an organization have accomplished while also seeing what we can improve: **What do you think we were able to accomplish in the event?** “We have solidified a partnership with an extremely influential tech incubator. Tech Square Labs created over 500 jobs and companies raised over $250 million of venture capital in one year. We exposed GSU students to more entrepreneurial opportunities and inspiration as Luis and Ahmed's company was in the final 5 (out of 250 companies) in the 1st Atlanta Startup Battle. This event put GSU and PantherHackers in a technologically relevant light that is usually only represented by GA Tech.  We are continuing to build a culture of hacking (creative problem solving). This event also continued building a combined culture with GSU Perimeter and GSU Downtown. 3/4 of the Potenza team are Perimeter Students. It was so great to give them a platform for sharing their skills and abilities.” **In what ways did you think that the event could be better the next time around?** “Do not remove people from the registration list, expand the list to 50% of the attendees we want. I will say I think that no showing is such a bad culture to accept. I am a believer in "do what you say you are going to do" so all the students who did not show up wasted their money. This event was for them paid by tech fee, their tech fees. We expected 70 after the event list was updated and only 30 showed up. It still was a great quality event and the students who came benefited and will have some incredible opportunities.” **Do you want to say anything for the PantherHackers/ attendees who were at PantherHacks?** “I want to say that I appreciate all their hard work. I want them to keep building, solving complex problems, building businesses and impacting the community. I also want to see them participating in these events because they are some of the top students related to technology and I want them to have a broader viewpoint of events and opportunities. They are pioneers at GSU and the culture they are creating is impacting the overall GSU image more than they might know.” On the behalf of all PantherHackers, I would like to say thank you to everyone who came to the event and made it a hackathon to remember! It was truly a great experience and let’s hope we can do this again soon. Congratulations to all the teams that participated, and a special thanks to all of our sponsors!  That’s it! As always: stay hungry! Riyan Imam
```

Comment:
I don't think having the whole article in a single line is a good idea @Macdonny18